src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityWrappers.cs

```diff
@@ -301,6 +302,13 @@ private static int AccessibilityGetCursorOffsetWrapper(IntPtr self)
             return GetViewFromRefObject(self).AccessibilityGetCursorOffset();
         }
 
+        private static IntPtr AccessibilityGetRangeExtentsWrapper(IntPtr self, int startOffset, int endOffset, int coordType)
+        {
+            Rectangle rect = GetViewFromRefObject(self).AccessibilityGetRangeExtents(startOffset, endOffset, (AccessibilityCoordinateType)coordType);
```

Comment:
`Rectangle` is `Disposable` so I believe this should be `using Rectangle rect = ...`.

Reply:
Yes true, it is Disposable. I have put the 'using'. Thank you for kind review again!